gcn_classic_to_json/notices/GECAM_FLT/__init__.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+from ..GECAM_GND import parse as parse_gecam_gnd
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[27:39]  # Unused. According to docs: '32 bytes for the future'
+    bin[21]  # Temporarily Unused. This is trigger_duration; currently no keyword
+    bin[22:24]  # Unused. Equivalent to rate_energy_range
+
+    detector_options = ["on", "triggered"]
+    detector_bits = np.flip(np.unpackbits(bin[26:27].view(dtype="u1")))
+    detector_status = [detector_options[bit] for bit in detector_bits[:25]]
```

Comment:
This field is called `trigger_dets` in the GCN Classic documentation, but I don't see any details about its structure. How did you work out this encoding?

Reply:
The source code had the following comment:

23   TRIGGER_DETS:          hexa-decimal                      <flight trigger only, 25 bits(1's & 0's), the trigger status of the 25 GRD detectors, 0=NO or 1=YES>